app/assets/javascripts/application.js

```diff
@@ -5,5 +5,7 @@
   directives to concatenate multiple Javascript files into one.
 */
 //= require ../../../node_modules/digitalmarketplace-govuk-frontend/govuk-frontend/all.js
+//= require ../../../node_modules/digitalmarketplace-govuk-frontend/digitalmarketplace/digitalmarketplace-govuk-frontend.js
 
 GOVUKFrontend.initAll()
+DMGOVUKFrontend.initAll()
```

Comment:
If we want to delay the rollout of the cookie banner/analytics until all the other frontend apps include and are ready for the banner, all we need to do is comment this line out. The cookie banner is hidden by default and the javascript determines whether or not to show the component.